Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -11,7 +11,7 @@ file that was distributed with this source code.
 {% spaceless %}
 
     <input type="text" id="{{ id }}_autocomplete_input" value=""
-        {%- if read_only is defined and read_only %} readonly="readonly"{% endif -%}
```

Comment:
This is a BC break. `read_only` option is still used on Symfony <2.7.

You should instead add a `NEXT_MAJOR: remove` comment.